rtrlib/rtr_mgr.h

```diff
@@ -73,7 +73,7 @@ typedef void (*rtr_mgr_status_fp)(const struct rtr_mgr_group *, enum rtr_mgr_sta
 /**
  * Type alias \c pfx_for_each_fp for function signature: \code (*)(const struct pfx_record *pfx_record, void *data) \endcode
  */
-typedef void (*pfx_for_each_fp)(const struct pfx_record *pfx_record, void *data);
+//typedef void (*pfx_for_each_fp)(const struct pfx_record *pfx_record, void *data);
```

Comment:
Its a duplicate, already defined in [rtrlib/pfx/pfx.h ](https://github.com/rtrlib/rtrlib/blob/c97e0043df737254e67dd23da8264a3e9f363ccf/rtrlib/pfx/pfx.h#L88). However, you're right should be removed completely.